testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -631,6 +628,17 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
     createNodeTripleInternal(BitcoindVersion.V17, clientAccum)
   }
 
+  /**
+    * Returns a triple of [[org.bitcoins.rpc.client.v18.BitcoindV18RpcClient BitcoindV18RpcClient]]
+    * that are connected with some blocks in the chain
+    */
+  def createNodeTripleV18(
+                           clientAccum: RpcClientAccum = Vector.newBuilder
+                         )(implicit system: ActorSystem): Future[
+    (BitcoindV18RpcClient, BitcoindV18RpcClient, BitcoindV18RpcClient)] = {
+    createNodeTripleInternal(BitcoindVersion.V18, clientAccum)
+  }
```

Comment:
Can you file an issue for making this call work (so that a future PR can deal with it)? I believe it currently doesn't since `generate` is deprecated in v18